Resync css/css-scoping tests

keithamus opened the pull request:
#### d4e0087dd80a198568d7cac91b98577b9438a7a2
<pre>
Resync css/css-scoping tests
<a href="https://bugs.webkit.org/show_bug.cgi?id=281746">https://bugs.webkit.org/show_bug.cgi?id=281746</a>

Reviewed by Tim Nguyen.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/9e160bd08fcbf426e7d91791d97a038fb3a37545">https://github.com/web-platform-tests/wpt/commit/9e160bd08fcbf426e7d91791d97a038fb3a37545</a>

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/import-expectations.json:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-001-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-002-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-003-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-004-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-005-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-006-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-006.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-007-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-007.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-008-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-008.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-009-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-009.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-001.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-001.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-002.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-002.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-003.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-003.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-changing-001.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-changing-001.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-changing-002.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-changing-002.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-flattened-001.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-flattened-001.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-flattened-002.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-flattened-002.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-flattened-003.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-flattened-003.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-001.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-001.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-002.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-002.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-003.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-003.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-004.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-004.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-005.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-005.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-006.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-006.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-007.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-007.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-changing-001.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-changing-001.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-changing-002.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-changing-002.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-changing-003.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-changing-003.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-changing-004.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-changing-004.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-flattened-001.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-flattened-001.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-flattened-002.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-flattened-002.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-flattened-003.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-flattened-003.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-flattened-004.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-flattened-004.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-flattened-005.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-flattened-005.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-flattened-006.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-flattened-006.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-defined-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-defined.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-has-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-has-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-has-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-has-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-has-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-has-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-has-internal-001.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-has-internal-001.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-has-internal-002.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-has-internal-002.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-has-internal-003.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-has-internal-003.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-is-featureless-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-is-featureless.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/keyframes-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/keyframes-002-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/keyframes-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/reference/green-box.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/reference/green-text.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/reference/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/shadow-reassign-dynamic-005-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/shadow-reassign-dynamic-006-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/shadow-reassign-dynamic-006-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/shadow-reassign-dynamic-006.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/w3c-import.log:

Canonical link: <a href="https://commits.webkit.org/286249@main">https://commits.webkit.org/286249@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/a2c7ba267585ab422cbce86821fe19d827f04dbe

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/75238 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/54677 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/28078 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/79699 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/26494 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/63817 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/2462 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/59050 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/17292 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/78305 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/49225 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/64630 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/39425 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/46607 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/22126 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/24822 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/67658 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/22466 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/81184 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/2568 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/1593 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/67300 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/2718 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/64630 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/66591 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/16580 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/10535 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/8699 "Passed tests") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/2529 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/2554 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/3483 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/2563 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (approved):
r=me once everything is green

keithamus committed:
Resync css/css-scoping tests
https://bugs.webkit.org/show_bug.cgi?id=281746

Reviewed by Tim Nguyen.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/9e160bd08fcbf426e7d91791d97a038fb3a37545

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/import-expectations.json:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-001-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-002-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-003-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-004-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-005-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-006-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-006.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-007-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-007.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-008-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-008.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-009-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/font-face-009.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-001.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-001.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-002.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-002.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-003.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-003.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-changing-001.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-changing-001.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-changing-002.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-changing-002.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-flattened-001.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-flattened-001.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-flattened-002.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-flattened-002.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-flattened-003.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-flattened-003.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-001.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-001.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-002.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-002.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-003.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-003.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-004.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-004.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-005.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-005.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-006.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-006.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-007.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-007.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-changing-001.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-changing-001.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-changing-002.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-changing-002.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-changing-003.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-changing-003.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-changing-004.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-changing-004.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-flattened-001.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-flattened-001.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-flattened-002.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-flattened-002.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-flattened-003.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-flattened-003.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-flattened-004.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-flattened-004.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-flattened-005.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-flattened-005.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-flattened-006.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/has-slotted-functional-flattened-006.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-defined-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-defined.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-has-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-has-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-has-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-has-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-has-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-has-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-has-internal-001.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-has-internal-001.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-has-internal-002.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-has-internal-002.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-has-internal-003.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-has-internal-003.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-is-featureless-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-is-featureless.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/keyframes-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/keyframes-002-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/keyframes-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/reference/green-box.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/reference/green-text.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/reference/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/shadow-reassign-dynamic-005-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/shadow-reassign-dynamic-006-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/shadow-reassign-dynamic-006-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/shadow-reassign-dynamic-006.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/w3c-import.log:

Canonical link: https://commits.webkit.org/286249@main
